test/smoke/multi-image3/Makefile

```diff
@@ -15,4 +15,4 @@ CC           = $(OMP_BIN) $(VERBOSE)
 include ../Makefile.rules
 run: $(TESTNAME)
```

Comment:
The `run` rule is not used by the `check_smoke` script, so, this extra part will not be tested.

Please use the `RUNCMD` to provide more complex rules than `./$(TESTNAME)`. The `RUNCMD` will be used by the `check` rule when set.

Reply:
made the change as requested :)